docs/try-viam/rover-resources/rover-tutorial-fragments.md

```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
 
 {{<imgproc src="try-viam/rover-resources/fragments/fragment_configuration.png" resize="1200x" alt="Fragment configuration">}}
 
-The fragment adds the following components to your robot's JSON configuration:
+The fragment adds the following components to your robot's JSON configuration.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component below.
```

Comment:
Avoid conditionals
```suggestion
If you would like to configure each component yourself, click the respective links on each component below.
```

Reply:
We already told them to add the fragment and save the config. I agree with moving these two sentences to below the list, but I think this change makes it more confusing?

Reply:
```suggestion
```